easybuild/easyconfigs/m/MariaDB/MariaDB-10.3.14-foss-2019a.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'CMakeMake'
+
+name = 'MariaDB'
+version = '10.3.14'
+
+homepage = 'https://mariadb.org/'
+description = """MariaDB is an enhanced, drop-in replacement for MySQL.
+Included engines: myISAM, Aria, InnoDB, RocksDB, TokuDB, OQGraph, Mroonga."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['http://ftp.hosteurope.de/mirror/archive.mariadb.org/mariadb-%(version)s/source']
```

Comment:
why use a mirror?

Reply:
There is no public hosting of sources at mariadb.org. The official download link for this package is `http://downloads.mariadb.org/interstitial/mariadb-10.3.14/source/mariadb-10.3.14.tar.gz` which automatically derives to one of the mirrors. `wget` is able to hop to the designated mirror but EB can't. So I used the same mirror found in previous easyconfigs of MariaDB.
The releases in Github lack features compared to the releases in mariadb.org, for instance the RocksDB engine is not there.